server/dash/dashboards/neighborhood_recent.py

```diff
@@ -66,6 +66,7 @@ def populate_options():
     y="srnumber",
     color="typeName",
     color_discrete_sequence=DISCRETE_COLORS,
+    color_discrete_map = DISCRETE_COLORS_MAP,
```

Comment:
nit: no spaces around "=" when using named arguments

Reply:
Resolve in newest commit